src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/AgencyResourceTest.kt

```diff
@@ -736,4 +738,20 @@ class AgencyResourceTest : ResourceTest() {
       AgencyEstablishmentType.builder().code("IM").description("Closed Young Offender Institute (Male)").build(),
     )
   }
+
+  @Test
+  fun testGetEventLocationsForAPrison() {
+    val token = authTokenHelper.getToken(AuthToken.REF_DATA_MAINTAINER)
```

Comment:
There wasn't even a test for this endpoint, I've added a very simple one, ideally it could do with more.